### DISCUSSION TITLE
Basic Insert-model for fusion 360 (.f3d)?

### DISCUSSION THREAD

**Discussion by Fenar91** [doudar/SmartSpin2k#394] [General]
Hello,
i want to model a new insert (for V2-Case) and need for that the basic model which i can use and edit with fusion. Unfortunatly i cant use .sldprt Files... 

Can anybody help me?

**Comment by doudar** [comment]
I'm on mobile right now so I can't verify this is 100% correct, but it looks like we might have one lone survivor .f3d insert file that you could use to base your design off of.

https://github.com/Hardware/Inserts/F3D_Designs/BH%2520SB3%2520spinbike%2520Insert%2520v2.f3d

**Reply by eMadman** [reply]
For what its worth, Fusion 360 CAN open .sldprt files, but I don't know if that's included with the free tier.  When you open the file, there's an option to "open from computer"

It will more or less convert it to a .step file and give you a great start point to work with.  It's how I've manipulated most of @doudar's CAD for the direct mount.

**Reply by Fenar91** [reply]
Thank u guys!!!
